cache/src/main/java/net/runelite/cache/NpcManager.java

```diff
@@ -97,14 +97,19 @@ public void java(File java) throws IOException
 		java.mkdirs();
 		try (IDClass ids = IDClass.create(java, "NpcID"))
 		{
-			for (NpcDefinition def : npcs.values())
+			try (IDClass nulls = IDClass.create(java, "NullNpcID"))
```

Comment:
For some reason the other classes don't do this, but you can have try-with-resources with multiple resources fyi.